packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.tsx

```diff
@@ -18,11 +19,12 @@ export const PaymentConsentCheckbox = ({
   plan,
   onClick,
 }: PaymentConsentCheckboxProps) => {
-  const { navigatorLanguages } = useContext(AppContext);
+  const { navigatorLanguages, config } = useContext(AppContext);
 
-  const { termsOfServiceURL, privacyNoticeURL } = productDetailsFromPlan(
-    plan,
-    navigatorLanguages
+  const { termsOfService, privacyNotice } = urlsFromProductConfig(
+    plan as AbbrevPlan,
```

Comment:
Instead of an assertion can we either a) update the `Plan` interface, or b) change `PaymentConfirmationProps` to use `AbbrevPlan` instead?

Reply:
My only concern in doing this here, and with the other components, is that I'll have to update a bunch of components.

I'd initially planned on updating `apiFetchPlans()` in `apiClient.ts` to return `Promise<AbbrevPlan[]>` instead of `Promise<Plan[]>`, but realized that would mean updating a bunch of dependencies.

What do you think about picking this up as a follow up ticket?

Reply:
How about option a) then?

Reply:
Option a) makes a lot of sense. I totally misread that at first, sorry about that.

I've made the suggested change! Thank you :)